### PR TITLE
Ensure that the file being processed has had its etag properly sanitised, log etag more

### PR DESCRIPTION
**Pull request by claucambra**
Might address https://github.com/nextcloud/server/issues/33785

@tobiasKaminsky 

Signed-off-by: Claudio Cambra <claudio.cambra@gmail.com>

### CODE REVIEW

File: src/libsync/discovery.cpp [hunk 1]
```diff
@@ -536,6 +543,11 @@ void ProcessDirectoryJob::processFileAnalyzeRemoteInfo(
                 item->_instruction = CSYNC_INSTRUCTION_NEW;
             } else {
                 item->_instruction = CSYNC_INSTRUCTION_SYNC;
+                qCInfo(lcDisco) << "CSYNC_INSTRUCTION_SYNC: File" << item->_file << "if (dbEntry._etag != serverEntry.etag)"
```
Comment (mgallien) [inline src/libsync/discovery.cpp:546]: maybe better use `qCDebug`

**Comment by tobiasKaminsky** [comment]
Thanks, I am testing it right now

File: src/libsync/discovery.cpp (hunk 1 above)
Comment (claucambra) [reply src/libsync/discovery.cpp]: Okay, changed all to qCDebug

**Comment by tobiasKaminsky** [comment]
This seems to be way better, but now "only" root one folder's etag keeps changing.
I can see also via web ui that modification date is always "a few seconds ago", while files/folders inside this folder have "13 days ago".
Files/folders next to the problematic folder, which are also synced, do not have a flipping eTag…

**Comment by tobiasKaminsky** [comment]
Hm. This seems to be unrelated to client, as it happens also in this case:
- note down etag: 123
- stop client
- refresh parent folder (where problematic folder is in) -> no change
- open folder via web
- see changing eTag

--> so it is enough to enter this folder to have a flipping etag

**Review by mgallien — APPROVED** [review]

**Pull request merged**